__tests__/data/firehose.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
If you want to commit this file please provide data too.

Reply:
done